pandas/core/reshape/pivot.py

```diff
@@ -493,6 +494,13 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
     rownames = _get_names(index, rownames, prefix='row')
     colnames = _get_names(columns, colnames, prefix='col')
 
+    obs_idxes = [obj.index for objs in (index, columns) for obj in objs
```

Comment:
I think might be nicer to fix ``_get_combined_index`` to ignore ``None`` in the array